### PR TITLE
Switch to DIV for swaps dropdown accessibility

### PR DESCRIPTION
**Pull request by darkwing**
Before:

Before

After:

After

### CODE REVIEW

**Review by Gudahtt — DISMISSED** [review]
LGTM!

File: ui/app/pages/swaps/build-quote/index.scss [hunk 1]
```diff
@@ -123,4 +123,9 @@
   .dropdown-input-pair__to .dropdown-search-list {
     width: 100%;
   }
+
+  .dropdown-input-pair__to  .dropdown-search-list__closed-primary-label {
```
Comment (Gudahtt) [inline ui/app/pages/swaps/build-quote/index.scss:127]: The linter is failing - possibly due to this extra space here?

File: ui/app/pages/swaps/dropdown-search-list/index.scss [hunk 2]
```diff
@@ -1,7 +1,5 @@
 .dropdown-search-list {
   flex-flow: column;
```
Comment (Gudahtt) [inline ui/app/pages/swaps/dropdown-search-list/index.scss:2]: Did you find out what this rule was intended to do after?

**Comment by darkwing** [comment]
Removed unnecessary styles thanks to Firefox;

Rule

File: ui/app/pages/swaps/dropdown-search-list/index.scss (hunk 2 above)
Comment (darkwing) [reply ui/app/pages/swaps/dropdown-search-list/index.scss]: Removed thanks to Firefox!

**Review by Gudahtt — APPROVED** [review]
LGTM! Tested on Chrome on fullscreen and in the popup, and I've tested escaping and selecting from the list, and navigating with tab. Everything seems to work.

**Pull request merged**